### PR TITLE
RemoteState

### PR DESCRIPTION
**Pull request by amitnayak1505**

### CODE REVIEW

File: output.tf [hunk 1]
```diff
@@ -1,3 +1,4 @@
 output "web_instance_ip" {
     value = aws_instance.public.public_ip
 }
+
```
Comment (vladimirmukhin) [inline output.tf]: This extra newline is not needed, please remove it

File: variables.tf [hunk 2]
```diff
@@ -7,3 +7,4 @@ variable vpc_cidr {}
 variable private_cidr {}
 
 variable public_cidr {}
+
```
Comment (vladimirmukhin) [inline variables.tf]: This extra newline is not needed, please remove it

**Review by vladimirmukhin — CHANGES_REQUESTED** [review]

File: output.tf (hunk 1 above)
Comment (amitnayak1505) [reply output.tf]: Updated removed the extra line

File: variables.tf (hunk 2 above)
Comment (amitnayak1505) [reply variables.tf]: Updated removed the extra line

**Pull request merged**